app/components/IntroScreen.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import { Box } from '@mui/material';
+import bg from '@/public/assets/images/start-screen-image.png';
+import Title from '@/app/components/Title';
+import LanguageModal from '@/app/components/LanguageModal';
+import { useState } from 'react';
+import { locales } from '@/middleware';
+import { AppRouterInstance } from 'next/dist/shared/lib/app-router-context.shared-runtime';
+import { getCookie, setCookie } from 'cookies-next';
+
+interface IntroScreenProps {
+  dict: Dict;
+  router: AppRouterInstance;
+}
+
+const IntroScreen = ({ dict, router }: IntroScreenProps) => {
+  const [isModalOpen, setIsModalOpen] = useState(!getCookie('language'));
+
+  const handleChangeLanguage = (localeString: Lang) => {
+    if (locales.includes(localeString)) {
```

Comment:
This is redundant now since `localeString` is of type `Lang`

Reply:
Woops, that slipped past me